server/graphql/v2/mutation/ExpenseMutations.ts

```diff
@@ -455,6 +455,11 @@ const expenseMutations = {
         type: new GraphQLNonNull(GraphQLAccountReferenceInput),
         description: 'Account where the expense will be created',
       },
+      skipInvite: {
+        type: GraphQLBoolean,
```

Comment:
Given this has a default value and we later check it with `!args.skipInvite`, I'd make it non-nullable otherwise passing `null` would act as `false`

```suggestion
        type: new GraphQLNonNull(GraphQLBoolean),
```